src/storm/transformer/SubsystemBuilder.cpp

```diff
@@ -134,6 +134,10 @@ SubsystemBuilderReturnType<ValueType, RewardModelType> internalBuildSubsystem(st
     } else {
         components.transitionMatrix = originalModel.getTransitionMatrix().getSubmatrix(false, keptActions, subsystemStates);
     }
+    if (options.makeRowGroupingTrivial) {
```

Comment:
Do we have to catch the case where the transitionMatrix is still supposed to be nondeterministic (i.e. not square)?

Reply:
Yes, indeed, will do!